google-cloud-datastore/src/test/java/com/google/cloud/datastore/testing/ITLocalDatastoreHelperTest.java

```diff
@@ -43,6 +46,12 @@
   private static final double TOLERANCE = 0.00001;
   private static final String PROJECT_ID_PREFIX = "test-project-";
   private static final String NAMESPACE = "namespace";
+  private Path dataDir;
+
+  @Before
+  public void setUp() throws IOException {
+    dataDir = Files.createTempDirectory("gcd");
```

Comment:
Looks good, can you also add a delete in an `@After` so we're not leaving lots of directories around?

Reply:
done